src/bcf/header.rs

```diff
@@ -110,6 +110,23 @@ impl Drop for Header {
 }
 
 
+/// A header record.
+pub enum HeaderRecord {
+    /// A `FILTER` header record.
+    Filter { key: String, key_value_pairs: Vec<(String, String)> },
```

Comment:
I think these should be `<type> {key: String, values: HashMap<String, String>}`, except for Generic.

Reply:
Wait, this should be LinearMap.